activerecord/CHANGELOG.md

```diff
@@ -425,6 +425,12 @@
     Previously this method always returned an empty array.
 
     *Kevin McPhillips*
+    
+*   Fix inconsistency when using scope name same as instance method of `ActiveRecord::Relation` class.
```

Comment:
"Fix inconsistency" sounds like we made this work correctly, when really we don't allow it any more. How about:

> Don't allow scopes to be defined which conflict with instance methods on `Relation`.

New changelog entries should be added at the top of the file.

Reply:
sounds good